examples/llm/src/models/mosaic_gpt.py

```diff
@@ -132,11 +136,38 @@ def _attn_bias(self, device, dtype):
 
         return self.attn_bias
 
+    def _apply_prefix_mask(self, attn_bias: torch.Tensor,
+                           prefix_mask: torch.Tensor):
+        assert attn_bias.shape[-1] == self.config.max_seq_len
```

Comment:
raise not assert.

also shouldn't it be `attn_bias.shape[-1] == self.config.max_seq_len and attn_bias.shape[-2] == self.config.max_seq_len`

Reply:
Since this method should only trigger for a prefix_lm, yes. Good catch!